src/item/ItemFactory.ts

```diff
@@ -22,6 +23,15 @@ export class ItemFactory<X extends Content> {
 
   constructor(private item: LocalItem | null = null) {}
 
+  /**
+   * Instantiates a new item with the base properties.
+   * @param id - The item's id.
+   * @param name - The item's name.
+   * @param rarity - The item's rarity.
+   * @param category - The item's category.
+   * @param collectionId - The id of the collection the item belongs to.
+   * @param description - The description of the item.
```

Comment:
What do you think about changing this to an object, to avoid having problems with partial arguments (that must be at the end) and avoid having a long list of params?